script/ts/commands/RebalanceHLP/execute-reinvest.ts

```diff
@@ -0,0 +1,58 @@
+import { Command } from "commander";
+import { loadConfig } from "../../utils/config";
+import signers from "../../entities/signers";
+import { RebalanceHLPHandler__factory } from "../../../../typechain";
+import { getUpdatePriceData } from "../../utils/price";
+import { ecoPythPriceFeedIdsByIndex } from "../../constants/eco-pyth-index";
+import chains from "../../entities/chains";
+import { Address } from "wagmi";
+
+type AddGlpParams = {
+  token: Address;
+  amount: number;
+  minAmountOutUSD: number;
+  minAmountOutGlp: number;
+};
+
+async function main(chainId: number) {
+  const config = loadConfig(chainId);
+  const provider = chains[chainId].jsonRpcProvider;
+  const deployer = signers.deployer(chainId);
+
+  const [readableTable, minPublishedTime, priceUpdateData, publishTimeDiffUpdateData, hashedVaas] =
+    await getUpdatePriceData(ecoPythPriceFeedIdsByIndex, provider);
+
```

Comment:
This touches critical part of the protocol.
So before submit the price please print them out like what's in feed price script and ask for a confirmation from the executor first.